<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookClassifyManager</class>
 <widget class="QWidget" name="BookClassifyManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>801</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="Title">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>  图书分类管理</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ReturnpushButton">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="DataWidget" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>911</width>
     <height>711</height>
    </rect>
   </property>
   <widget class="QLabel" name="BookInformationManagerTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>  图书分类管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddpushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>添加➕</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>871</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SurepushButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>670</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>确认✔</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CanclepushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>670</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>删除❌</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Manager">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>991</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>                                                                                                                                                                                                                                        管理员：gkb</string>
   </property>
  </widget>
  <widget class="Line" name="line1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>991</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
